src/pylon_camera/pylon_camera_node.cpp

```diff
@@ -178,14 +222,13 @@ bool PylonCameraNode::startGrabbing()
     for ( int i = 0; i < set_user_output_srvs_.size(); ++i )
     {
         std::string srv_name = "set_user_output_" + std::to_string(i);
-        set_user_output_srvs_.at(i) = nh_private_->advertiseService< camera_control_msgs::SetBool::Request,
-                                                            camera_control_msgs::SetBool::Response >(
-                                            srv_name,
-                                            boost::bind(&PylonCameraNode::setUserOutputCB,
-                                                        this,
-                                                        i,
-                                                        _1,
-                                                        _2));
+        set_user_output_srvs_.at(i) = nh_private_->advertiseService< camera_control_msgs::SetBool::Request, 
+                                                                     camera_control_msgs::SetBool::Response >( 
```

Comment:
Actually, screw backwards compatibility in this case. Can you just get rid of the individual parameter setting services? They are obsolete once we can change things with dynamic_reconfigure.